hooks/tk-3dsmax_actions.py

```diff
@@ -78,7 +78,39 @@ def generate_actions(self, sg_publish_data, actions, ui_area):
                                       "description": "This will insert a reference to this file into the current scene."} )        
         
         return action_instances
-                
+
+    def execute_action_on_selection(self, name, action_params):
```

Comment:
This is the backwards compatible code that is added for every DCC. This should be copied in the Katana hook.